### ISSUE TITLE
Invalid octal literals

### ISSUE THREAD

**Issue by glitch003** [mhr3/unzip-stream#12]
Hi, when packaging up unzip-stream into my bundle, I got an error that the octal literals are an "invalid number" and will only work in non-strict mode.  I fixed this by changing them from `012` to `0o12` which is the es6 format for octal: https://github.com/lukehoban/es6features#binary-and-octal-literals

```
Acorn error: Invalid number (24946:40)
File: /Users/chris/Documents/WorkStuff/Deconet/Frontend/build/index.bundle.js

24944                  unixAttrs = this.parsedEntity.externalFileAttributes >>> 16;
24945                  var fileType = unixAttrs >>> 12;
24946                  isSymlink = (fileType & 012) === 012; // __S_IFLNK
24947              }
24948              if (this.options.debug) {
24949                  const debugObj = Object.assign({}, this.parsedEntity, {
24950                      path: path,
```

Pull request incoming :)

**Comment by glitch003** [comment]
PR is here: https://github.com/mhr3/unzip-stream/pull/13

Thanks!

**Comment by mhr3** [comment]
Fixed in #11

**Comment by glitch003** [comment]
Yay thanks!
